packages/@eris-boiler/common/src/modules/stores/load-map.ts

```diff
@@ -35,6 +45,30 @@ export abstract class LoadMap<T> extends ExtendedMap<string, T> {
     return this
   }
 
+  public async reload (): Promise<this> {
+    const reloadPaths = Object.keys(this.reloadables)
+    for (const reloadable of reloadPaths) {
```

Comment:
Run a benchmark. Check if it's faster to use `for ... in` and push each one to `this.toLoad`.

Reply:
why `for in` over `for of`? and lastly I think they're around the same speed so there's not much reason why to-do so...